versions.tf

```diff
@@ -1,4 +1,5 @@
 terraform {
+  experiments = [module_variable_optional_attrs]
```

Comment:
This feature was added in terraform version 1.3. As we don't set a minimum terraform version, it seems it's using a version <1.3 so we need to explicit specific the feature